Examples/SwiftUIMatchExample.swift

```diff
@@ -0,0 +1,90 @@
+import UIKit
+import SwiftUI
```

Comment:
Can you wrap this while class in, 
```swift
#if canImport(SwiftUI)
...
#endif
```

So that persons using older versions of XCode don't receive build errors.

You may need to wrap other code in the same block if they reference these classes.

Reply:
Sure, will do that tomorrow 👌

Reply:
I've done what you suggested, however the checks related to that are still failing. I have no clue why that is. Hope you do.

I will still attempt to fix this one:
<img width="1274" alt="image" src="https://user-images.githubusercontent.com/2656075/69039663-a4cc3680-09ec-11ea-981c-d672a0730346.png">

Reply:
Ok, that last check is fixed now. Clueless on how to improve this code now. Looking forward to hearing an update from you @JoeMatt

Reply:
@JoeMatt do you have any updates for me on how we can get this merged?